database/migrations/2019_10_23_142210_create_faults_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFaultsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('faults', function (Blueprint $table) {
```

Comment:
The location, the reporter and the report dates could also be added here.

Reply:
Added location and reporter id, date has always been there (`$table->timestamps();`).